prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0, name='Car'):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self._odometer = 0
+        self.name = name
+
+    def __str__(self):
+        return f"{self.name}, fuel={self.fuel}, odometer={self._odometer}"
+
+    def add_fuel(self, amount):
+        """Add amount to the car's fuel."""
+        self.fuel += amount
+
+    def drive(self, distance):
+        """Drive the car a given distance.
+
+        Drive given distance if car has enough fuel
+        or drive until fuel runs out return the distance actually driven.
```

Comment:
Sentence structure here isn't amazing. It gets the point across but does it read very well?

Reply:
Thanks for the feedback, I believe it is like that in the template though.